front-end/src/global/statuses.ts

```diff
@@ -27,3 +27,10 @@ export const motionStatus = {
 	PROPOSED: 'Proposed',
 	VOTED: 'Voted'
 };
+
+export const tipStatus = {
```

Comment:
Here we can have something like:
```
const tipStatus = {
  CLOSED: {chain: 'TipClosed', human: 'Closed'},
  CLOSING: {chain: 'TipClosing', human: 'Closing' },
...
}
```

Reply:
ok, to stay consistent here, this is actually better. We can then add a
```js
const tipStatusDisplay = {
  'TipClosed': 'Closed',
  'TipOpen': 'Open'
...
}
```

and user this in the UI.

Reply:
This cannot be done. Either use Open Closed at prisma or show TipClosed etc. Too many typescript error